helper.py

```diff
@@ -944,19 +947,15 @@ def map_source_receiving_site_address(_source_sites, _receiving_sites):
 def get_regional_district(_lat, _long):
     """Returns regional district for latitude and longutide"""
     _arcgis_regional_districts_query_url = FEATURE_SERVICE_URL + '&geometry=' + str(_long) + ',' + str(_lat)
-    _service_response = requests.request("GET", FEATURE_SERVICE_URL)
-    _service_response_json = json.loads(_service_response.content)
-    if _service_response_json.get('features')[0].get('attributes').get('ADMIN_AREA_NAME'):
-        return _service_response_json['features'][0]['attributes']['ADMIN_AREA_NAME']
-    else:
-        raise KeyError("status code:" + str(_service_response.status_code))
-    """
-    if _ches_response.status_code == 200:
-        logging.info(constant.CHES_HEALTH_200_STATUS)
-    elif _ches_response.status_code == 401:
-        logging.error(constant.CHES_HEALTH_401_STATUS)
-    elif _ches_response.status_code == 403:
-        logging.error(constant.CHES_HEALTH_403_STATUS)
-    else:
-        logging.error("CHES Health returned staus code:%s, text:%s", str(_ches_response.status_code), _ches_response.text)
-    """        
\ No newline at end of file
+    try:
+        _service_response = requests.request("GET", FEATURE_SERVICE_URL, timeout=5)
```

Comment:
could we rename the `timeout` to be more precise like if it is in millis `timeoutInMillis` or if seconds `timeoutInSeconds`

Reply:
The 'timeout' is fixed parameter name in requests.request. We can't change it. Instead, I put a comment beside of that shows it is timeout in seconds.
`_service_response = requests.request("GET", FEATURE_SERVICE_URL, timeout=5) # timeout in seconds`

Reply:
thanks that helps :)